ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/eos_cli_config_gen.schema.yml

```
@@ -334,6 +334,11 @@ keys:
                 type: str
                 display_name: Regular Expression
                 description: If defined, a regex community-list will be configured
+  ip_dhcp_relay:
```

We need the display name for acronyms.
```suggestion
  ip_dhcp_relay:
    display_name: IP DHCP Relay
```